data/scripts/creaturescripts/player/update_client_on_advance_skill.lua

```diff
@@ -1,7 +1,7 @@
 local updateClientOnAdvanceSkill = CreatureEvent("Update Client On Advance Skill")
 
 function updateClientOnAdvanceSkill.onAdvance(player, skill, oldLevel, newLevel)
-	if skill == SKILL_LEVEL then
+	if skill == SKILL_LEVEL and newLevel > oldLevel then
```

Comment:
Though, why the `and newLevel > oldLevel` here? This script will now take a screenshot if the player losses a level.